### PR TITLE
Judge detail itegration into heairngs api

### PR DESCRIPTION
**Pull request by hmcts-shajahan**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/fis-hmc-api/values.yaml [hunk 1]
```diff
@@ -38,3 +38,4 @@ java:
     PRL_URL: 'http://prl-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal'
     PRL_HMC_S2S_AUTHORISED_SERVICES : xui_webapp,prl_cos_api,fis_hmc_api
     REF_DATA_URL: 'http://rd-location-ref-api-demo.service.core-compute-demo.internal/'
+    REF_DATA_JUDICIAL_URL: 'http://rd-judicial-api-demo.service.core-compute-demo.internal/'
```
Comment (hmcts-kodatala) [inline charts/fis-hmc-api/values.yaml:41]: we need to change based on environments istead of only demo

**Review by hmcts-kodatala — CHANGES_REQUESTED** [review]
pls check commnets

File: charts/fis-hmc-api/values.yaml (hunk 1 above)
Comment (hmcts-shajahan) [reply charts/fis-hmc-api/values.yaml]: Done

**Review by hmcts-kodatala — APPROVED** [review]
Ok

**Pull request merged**